packages/suite/src/components/suite/Preloader/SuiteLayout/Sidebar/QuickActions.tsx

```diff
@@ -76,48 +79,76 @@ const TorToggleContainer = styled.div`
 export const QuickActions = () => {
     const isDiscreetModeActive = useSelector(selectIsDiscreteModeActive);
     const { isTorEnabled, isTorLoading } = useSelector(selectTorState);
+    const enabledNetworks = useSelector(state => state.wallet.settings.enabledNetworks);
 
     const dispatch = useDispatch();
     const { translationString } = useTranslation();
     const theme = useTheme();
 
     const handleDicreetModeClick = () => dispatch(setDiscreetMode(!isDiscreetModeActive));
-
-    if (!isDesktop()) {
-        const translationLabel = isDiscreetModeActive ? 'TR_SHOW_BALANCES' : 'TR_HIDE_BALANCES';
-
-        return (
-            <WebContainer onClick={handleDicreetModeClick}>
-                <DescreetContainer>
-                    <Icon size={16} icon={isDiscreetModeActive ? 'HIDE' : 'SHOW'} />
-                    <Label>{translationString(translationLabel)} </Label>
-                </DescreetContainer>
-            </WebContainer>
-        );
-    }
+    const customBackends = useCustomBackends().filter(backend =>
+        enabledNetworks.includes(backend.coin),
+    );
```

Comment:
nit: seeing `filter()` right after the hook is a bit confusing, at least was not seeing that often. IMO, having a hook on it's own line clearly highlights hook usage, which usually implies some useEffects / refs and other hook-specific things.